api/osf_api.py

```diff
@@ -883,3 +909,48 @@ def delete_registration_version_draft(session, draft_id):
         session.api_base_url, draft_id
     )
     session.delete(url=registration_version_url, item_type='schema-responses')
+
+
+def submit_project_to_collection(
+    session, collection_guid, node_id, collected_type='selenium'
+):
+    """Submit a given project node to a collection.  This applies to branded public
+    collections and not a user's private custom collection.  NOTE: This function will
+    only work for collections using the 'collected_type' attribute.  Some collections
+    use different attributes (ex: 'school_type', 'study_design', 'status', etc.). If
+    this function will be used in the future to submit to a collection that uses one
+    of the other attributes, then the raw_payload below will need to be updated to add
+    those specific attributes.
+    """
+    if not session:
+        session = get_default_session()
+    user = current_user(session)
+
+    url = '/v2/collections/{}/collection_submissions/'.format(collection_guid)
+
+    raw_payload = {
+        'data': {
+            'type': 'collection_submissions',
+            'attributes': {
+                'collected_type': collected_type,
+                'guid': node_id,
+            },
+            'relationships': {
+                'collection': {
+                    'data': {
+                        'id': collection_guid,
+                        'type': 'collections',
+                    }
+                },
+                'creator': {
+                    'data': {
+                        'id': user.id,
+                        'type': 'users',
+                    }
+                },
+            },
+        },
+    }
+    session.post(
+        url=url, item_type='collection_submissions', raw_body=json.dumps(raw_payload)
+    )
```

Comment:
As discussed, return the response in case we need it in future.

In addition, there is no need to handle request failures since it is a well controlled test here but could use a comment that we don't do it.

Reply:
Thanks!. Fixed.